src/AbstractOperations/conditional_operations.jl

```diff
@@ -123,11 +123,11 @@ end
 @inline get_condition(condition::AbstractArray, i, j, k, grid, args...) = @inbounds condition[i, j, k]
 
 
-Base.summary(c::ConditionalOperation) = string("ConditionalOperand of ", summary(c.operand), " with condition ", summary(c.condition))
+Base.summary(c::ConditionalOperation) = string("ConditionalOperation of ", summary(c.operand), " with condition ", summary(c.condition))
 
 Base.show(io::IO, operation::ConditionalOperation) =
     print(io,
-          summary(operation), '\n',
+          "ConditionalOperation at $(location(operation))", '\n',
```

Comment:
why?

Reply:
Just to avoid a bit of repetition

Reply:
I'd argue that rewriting part of `summary` in `show` *is* repetition :-0

Reply:
but don't worry about it